src/Tools/ResponseStrategies/ResponseFileStrategy.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Mpociot\ApiDoc\Tools\ResponseStrategies;
+
+use Illuminate\Routing\Route;
+use Mpociot\Reflection\DocBlock\Tag;
+
+/**
+ * Get a response from from a file in the docblock ( @responseFile ).
+ */
+class ResponseFileStrategy
+{
+    public function __invoke(Route $route, array $tags, array $routeProps)
+    {
+        return $this->getFileResponse($tags);
+    }
+
+    /**
+     * Get the response from the file if available.
+     *
+     * @param array $tags
+     *
+     * @return mixed
+     */
+    protected function getFileResponse(array $tags)
```

Comment:
Code here is the same as that for `ResponseTagStrategy`. I'm assuming this PR is still a Work in Progress?

Reply:
I'll do some changes, and create the tests.

Reply:
It is different, in that method I read the filename and get the content

Reply:
Oh, sorry. I missed that. Please rename the variables so it's easier to see (see my suggestions below)L